python/src/scipp/io/hdf5.py

```diff
@@ -0,0 +1,102 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @file
+# @author Simon Heybrock
+
+
+def _unit_lut():
+    from .._scipp.core import units as u
+    units = [u.one, u.m, u.kg, u.us, u.angstrom]
```

Comment:
Aren't we missing lots of units? e.g. all the different combinations such as `m/s`.
What about `counts`?